build-system/tasks/extension-helpers.js

```diff
@@ -409,14 +409,14 @@ async function buildExtension(
       return;
     }
   }
+  if (name === 'amp-analytics') {
+    await vendorConfigs(options);
+  }
   if (argv.single_pass) {
     return;
   } else {
     await buildExtensionJs(path, name, version, latestVersion, options);
   }
```

Comment:
Can you remove the `if... return` here? Not sure why we have this.

So just `if (!argv.single_pass) { await build... }`

Reply:
ha! Good catch, done.